### PR TITLE
fix(voice): harden local audio readiness gates

### PR DESCRIPTION
**Pull request by Balizero1987**
## Summary
- Add real deep warm checks for local `whisper.cpp` and Chatterbox provider readiness.
- Keep LiveKit optional in static readiness but block production promotion in deep mode until worker health is wired.
- Require a dedicated frontend voice bridge secret via `VOICE_CONCIERGE_BACKEND_API_KEY`; no fallback to broad `API_KEYS`.
- Tighten the cloud text-mode PII guard for bare person-name concierge requests.
- Update the local audio runbook and env example with the Pro/Mini runtime boundary.

## Privacy and Runtime Boundary
- Air-M5 remains fail-closed for heavy voice runtime work.
- Real Whisper/Chatterbox/LiveKit runtime checks are intended for Pro/Mini.
- No cloud audio fallback was added.
- Readiness failures are sanitized to avoid leaking local paths or raw exception details.

## Verification
- `apps/backend-rag`: broad pre-push suite: `15561 passed, 853 skipped, 1 warning in 273.34s`.
- `apps/backend-rag`: targeted voice/audio tests: `114 passed, 1 skipped in 35.12s`.
- `apps/mouth`: targeted voice concierge tests: `82 passed`.
- `apps/backend-rag`: targeted `ruff check`: passed.
- `apps/mouth`: `npm run typecheck`: passed.
- `apps/mouth`: `npm run lint`: passed with `0 errors, 144 warnings`.
- `git diff --check origin/main...HEAD`: passed.

## Remaining Production Gate
Before promoting the local voice runtime, run the deep doctor on Pro or Mini:

```bash
cd ~/Desktop/nuzantara/apps/backend-rag
source .venv/bin/activate
PYTHONPATH=. python scripts/local_audio_doctor.py --mode deep --json
```

Expected current behavior: provider warm checks execute; full production readiness remains blocked until LiveKit worker health is implemented.

### CODE REVIEW

**Comment by Balizero1987** [comment]
## ❓ Tri-LLM review: **INCONCLUSIVE** (3/3 live-green)

`diff_sha=ba4a7dcaae82eed8` · 11363 lines · diff_complete=False

> ⚠️ **Diff was truncated** — reviewers did not see the full change. Verdict forced to `inconclusive`; treat as NOT reviewed.

- ✅ **codex** → `green` — No P0/P1 issues found in the voice concierge changes. The diff avoids new OpenAI/Anthropic key paths, leaves embeddings/dependencies/migrations untouched, and keeps the new local-audio surface gated behind existing router/auth boundaries.
- ✅ **claude-opus** → `green` — Local-only voice concierge stack is well-gated (disabled-by-default config, host allowlist, auth on every route, bounded uploads, no raw-audio persistence) and clean on all hard rules — no ANTHROPIC_API_KEY/OPENAI_API_KEY, embedding model untouched, no httpx-in-loop, no print/secrets, lazy heavy imports in chatterbox.py. Two P1s to confirm pre-merge: the two referenced-but-unshown provider modules must use deferred imports, and the status route performs a blocking import in the async path. Note: only ~6 of 43 changed files were visible, so router_manifest/public_endpoints/tests could not be fully verified.
    - 🟠 P1: apps/backend-rag/backend/app/routers/voice.py:21-24 — voice.py imports SileroVADProvider (silero_vad.py) and WhisperCppSTTProvider (whisper_cpp.py) at module top-level, but neither file is present in the provided diff. Since voice.py is loaded at router-registration time, these modules MUST defer heavy deps (no top-level `import torch`/`silero_vad`/whisper bindings) exactly like chatterbox.py does via `_safe_find_spec`/lazy `importlib.import_module`; otherwise cloud prod router import crashes (ModuleNotFoundError). Verify before merge.
    - 🟠 P1: apps/backend-rag/backend/app/services/local_audio/chatterbox.py (`_runtime_import_error`/`status()`) — `ChatterboxTTSProvider.status()` calls `importlib.import_module(f'{module_name}.mtl_tts')` (transitively torch) synchronously, and it is reached from the async `GET /local-audio/status` handler. On approved hosts where deps are installed, the first call blocks the event loop for the full torch import (seconds). Probe via `asyncio.to_thread` or cache the import result.
- ✅ **deepseek** → `green` — No P0 issues: no hard-rule violations, no secret leaks, no embedding model change. Missing tests for critical voice concierge endpoints is the only P1.
    - 🟠 P1: apps/backend-rag/backend/app/routers/voice.py: new /local-audio/* endpoints without corresponding automated tests

---
_Automated tri-LLM review (Codex GPT-5.5 · Claude Opus · DeepSeek V4-Pro). Review-only — **merge is the operator's decision** (Legge 5). This bot never labels, approves, or merges._

**Comment by Balizero1987** [comment]
## ❓ Tri-LLM review: **INCONCLUSIVE** (3/3 live-green)

`diff_sha=3b6ea5a118591a92` · 13238 lines · diff_complete=False

> ⚠️ **Diff was truncated** — reviewers did not see the full change. Verdict forced to `inconclusive`; treat as NOT reviewed.

- ✅ **codex** → `green` — No P0 hard-rule violations found: no new Anthropic/OpenAI key path, no embedding change, no migration issue, and the router is already registered. Main concerns are scoped auth for sensitive audio routes and a flaky environment-dependent unit test.
    - 🟠 P1: apps/backend-rag/backend/app/routers/voice.py:50 — new local-audio STT/TTS endpoints reuse global settings.api_keys via verify_api_key(), so any existing service API key can access audio routes despite the docs/env contract requiring a dedicated voice bridge secret.
    - 🟠 P1: apps/backend-rag/backend/tests/unit/app/routers/test_voice_legacy.py:198 — test_local_audio_status_redacts_whisper_paths depends on ambient optional runtime availability; it failed in a local-audio-capable venv because turn_detection_ready became true, so the new test suite is nondeterministic.
- ✅ **claude-opus** → `green` — Feature voice-concierge local-audio ben isolata: gated da flag default-false + host allowlist Pro/Mini, nessuna violazione hard-rule (no ANTHROPIC_API_KEY/OPENAI_API_KEY, embedding intatto, no nuovo router da registrare, endpoint auth-protected quindi non-public, no httpx in-loop, no print, no secret hardcoded; I/O bloccante incapsulato in asyncio.to_thread e subprocess senza shell). Unico rilievo: import torch/chatterbox sincroni nel path async di /local-audio/status. NOTA: diff troncato — whisper_cpp.py, silero_vad.py, router_registration.py/router_manifest.py e gli eventuali test non sono visibili, quindi registrazione endpoint e coverage non verificati direttamente.
    - 🟠 P1: voice.py get_local_audio_status() / GET /local-audio/status endpoint — quando il flag è ON (Pro/Mini), i .status() dei provider eseguono import sincroni pesanti nell'event loop async: chatterbox.py _runtime_import_error() fa importlib.import_module('chatterbox.mtl_tts') (che tira dentro torch) + _safe_find_spec('torch'/'soundfile'), tutto fuori da asyncio.to_thread. Blocco event-loop per secondi sotto carico. Andrebbe spostato in asyncio.to_thread (impatto limitato: gated a host approvati + flag, cloud resta off).
- ✅ **deepseek** → `green` — The diff adds local-only audio concierge capabilities with proper isolation, no hard-rule violations, and no immediate production risks.

---
_Automated tri-LLM review (Codex GPT-5.5 · Claude Opus · DeepSeek V4-Pro). Review-only — **merge is the operator's decision** (Legge 5). This bot never labels, approves, or merges._
